frontend/src/Components/login.css

```diff
@@ -0,0 +1,138 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
+*{
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+}
+
+.login_form{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  background-color: #F8FFE5;
+}
+
+img {
+  width: 75px;
+  height: auto;
+  margin-bottom: 10px;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 32px;
+  text-align: center;
+  align-items: center;
+  letter-spacing: -0.015em;
+  color: #000000;
+  margin-bottom: 0px;
+}
+
+.subheader{
+  font-weight:lighter;
+  font-size: 18px;
+  text-align: center;
+  letter-spacing: -0.015em;
+  color: #000000;
+  margin-bottom: 40px;
+}
+
+form{
+  background:#FFFFFF;
+  width: 380px;
+  height: 220px;
+  padding: 35px;
+  border: 1px solid #D7D7D7;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 6px;
+  display: flex;
+  flex-direction: column;
+}
+
+.user_creds{
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  margin-bottom: 0%;
+}
+
+.email{
+  margin-bottom: 8px;
+}
+
+input{
```

Comment:
I'm purposefully going to ask us not to change this knowing that this might be problematic. This CSS rule will format all inputs, so it's possible this will mess w/ the sign up pages styling too.

Reply:
Okay but how would we get the input boxes to be the shape that Ray envisioned on figma?

Reply:
right, so this is also a TODO for @jesshe004 to install and configure css-modules! That will fix this issue.

Reply:
yeah for now though you can simply change this `input` selector into a new class, and then apply that class to each of the inputs on your login page

Reply:
Okay used their class names instead for styling.